modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -168,14 +168,14 @@ const augmentEditorUiApi = (editor: Editor, api: Partial<EditorUiApi>) => {
   editor.ui = { ...editor.ui, ...uiApiFacade };
 };
 
-const init = (editor: Editor): void => {
+const init = async (editor: Editor): Promise<void> => {
   editor.dispatch('ScriptsLoaded');
 
   initIcons(editor);
   initTheme(editor);
   initModel(editor);
   initPlugins(editor);
-  const renderInfo = renderThemeUi(editor);
+  const renderInfo = await renderThemeUi(editor);
```

Comment:
I would have expected the `renderThemeUi` to have some form of Promise check with branching and wrapping it in a promise, does await work on sync functions?

Reply:
Yes it does 😆

Reply:
Such a bizarre language.